run_ingestion/Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+# Use the official lightweight Python image.
+# https://hub.docker.com/_/python
+FROM python:3.8-slim
```

Comment:
Python 3.8 is beta in cloud functions, but I didn't see equivalent documentation for Cloud Run. Do you know if it's fully supported?

Reply:
Cloud Run doesn't have the same runtime restrictions since users do the build themselves.